op-wheel/cheat/cheat.go

```diff
@@ -78,12 +78,6 @@ func (ch *Cheater) Close() error {
 	return ch.DB.Close()
 }
 
-func openStorageTrie(s *state.StateDB, addr common.Address) (state.Trie, error) {
-	stateRoot := s.IntermediateRoot(true)
-	storageRoot := s.GetStorageRoot(addr)
-	return s.Database().OpenStorageTrie(stateRoot, addr, storageRoot)
-}
```

Comment:
Note for reviewers, added `OpenStorageTrie` in https://github.com/ethereum-optimism/op-geth/pull/215/commits/298900cf96512f9b4c3893e607b338112d62f7d1 (discussed with @ajsutton)